controller/config.yaml

```diff
@@ -0,0 +1,17 @@
+processor:
+  ignoreTypes:
+    - "(Elasticsearch|Kibana|ApmServer)List$"
+    - "(Elasticsearch|Kibana|ApmServer)Health$"
+    - "(Elasticsearch|Kibana|ApmServer|Reconciler)Status$"
+    - "ElasticsearchSettings$"
+    - "Associa(ted|tor|tionStatus|tionConf)$"
+  ignoreFields:
+    - "status$"
+    - "TypeMeta$"
+
+render:
+  kubernetesVersion: 1.22
+  knownTypes:
+    - name: SecretObjectReference
```

Comment:
probably should add ObjectReference and some other core types we use

Reply:
done